Create garbage collection docs (#28870)
kubernetes/website: content/en/docs/tasks/administer-cluster/use-cascading-deletion.md

    "apiVersion": "apps/v1",
    "metadata": {
        "name": "nginx-deployment",
        "namespace": "default",
        "uid": "d1ce1b02-cae8-4288-8a53-30e84d8fa505",
        "resourceVersion": "1363097",
        "creationTimestamp": "2021-07-08T20:24:37Z",
        "deletionTimestamp": "2021-07-08T20:27:39Z",
        "finalizers": [
          "foregroundDeletion"
        ]
        ...
   ```
   {{% /tab %}}
   {{</tabs>}}

## Use background cascading deletion

1. [Create a sample Deployment](/docs/tasks/run-application/run-stateless-application-deployment/#creating-and-exploring-an-nginx-deployment).
1. Use either `kubectl` or the Kubernetes API to delete the Deployment.

{{<tabs name="background_deletion">}}
{{% tab name="kubectl" %}}
Kubernetes uses background cascading deletion by default, and will do so
even if you run the following commands without the `--cascade` flag.

**Kubernetes version 1.20.x and later**

Run the following command:

```shell
kubectl delete deployment nginx-deployment --cascade=background
```

**Kubernetes versions prior to 1.20.x**

Run the following command:

```shell
kubectl delete deployment nginx-deployment --cascade=true

> ```

Reviewer: Having seen the preview, I'd prefer to make this Kubernetes version choice happen in tabs, and use headings to separate the `kubectl` vs. API + curl approaches.

The second tab can also refer readers to https://kubernetes.io/docs/home/supported-doc-versions/

Author: Okay. Can you confirm what the actual version number is? I wasn't sure from the commit (I've added a TODO comment in the MD to remind myself to verify)

Reviewer: I don't know when the change happened (I'd have to ask Google or, failing that, spelunk in the source code history for `kubectl`).

{{% /tab %}}
{{% tab name="API" %}}

> Kubernetes uses background cascading deletion by default, and will do so

Reviewer: nit:
```suggestion
Kubernetes uses background cascading deletion by default, and does so
```
